pkg/diagnostics/systemd/systemd.go

```diff
@@ -56,11 +57,26 @@ be the same for each.`,
 // captures for logMatcher Interpret functions to store state between matches
 var tlsClientErrorSeen map[string]bool
 
+type unitSpecArray []*unitSpec
+
+func (specs unitSpecArray) complete() unitSpecArray {
+	newSpecs := specs
+	for _, spec := range specs {
+		// analysis is same for origin or atomic-openshift journals
+		if strings.HasPrefix(spec.Name, "origin-") {
+			newSpec := *spec
+			newSpec.Name = strings.Replace(spec.Name, "origin-", "atomic-openshift-", 1)
+			newSpecs = append(newSpecs, &newSpec)
+		}
+	}
+	return newSpecs
+}
+
 // TODO, these should probably be split into interfaces similar to Diagnostic
 // Specify what units we can check and what to look for and say about it
-var unitLogSpecs = []*unitSpec{
+var unitLogSpecs = unitSpecArray{
 	{
-		Name:       "atomic-openshift-master",
+		Name:       "origin-master",
```

Comment:
I'd rather see this be `Names: sets.String("origin-master","atomic-openshift-master"),` than do name-fiddling elsewhere